frontend/src/patientApp/timeOfTest/constants.ts

```diff
@@ -37,6 +38,7 @@ export const respiratorySymptomsMap = {
   "422587007": "Nausea",
   "422400008": "Vomiting",
   "62315008": "Diarrhea",
+  "261665006": OTHER_SYMPTOM_NOT_LISTED_LITERAL,
```

Comment:
are we supporting Other for Covid only and not syphilis?

Reply:
I wanted to wait to put that in with the other syphilis UI code so I can generate the snapshot together.